src/NuGet.Clients/PackageManagement.UI/Feeds/MultiSourcePackageFeed.cs

```diff
@@ -151,12 +151,18 @@ public MultiSourcePackageFeed(IEnumerable<SourceRepository> sourceRepositories,
 
             if (notCompleted.Any())
             {
-                var items = notCompleted
-                        .ToDictionary(kv => kv.Key, kv => GetLoadingStatus(kv.Value.Status));
+                var items = notCompleted.ToDictionary(
+                    kv => kv.Key,
+                    kv => new { LoadingStatus = GetLoadingStatus(kv.Value.Status), kv.Value.Exception });
```

Comment:
stylistic mostly. performance - potentially because of that extra pack into the anonymous object. If there's a way to avoid it I think it should be prioritized. Does it make sense?